challenge3/webapp/y-shinada/app/models/house.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: houses
+#
+#  id            :bigint(8)        unsigned, not null, primary key
+#  firstname     :string(255)      not null
+#  lastname      :string(255)      not null
+#  city          :string(255)      not null
+#  num_of_people :integer          default(1), unsigned, not null
+#  has_child     :boolean          default(FALSE), not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_houses_on_city  (city)
+#
+
+class House < ApplicationRecord
+  scope :with_city, ->(c = "") { !c.blank? ? where(city: c) : self } # rubocop:disable Style/TernaryParentheses
```

Comment:
! .blank? とするよりは .present? を使う方が好ましいと感じた。

```suggestion
  scope :with_city, ->(c = "") { c.present? ? where(city: c) : self }
```

Reply:
これはミスでした 🙇 
その時に `presence` と混同していたのかもしれません。